api/v1alpha1/conditions_const.go

```diff
@@ -36,17 +36,28 @@ const (
 )
 
 const (
-	// HivelocityClusterReady reports on whether the Hivelocity cluster is in ready state.
-	HivelocityClusterReady clusterv1.ConditionType = "HivelocityClusterReady"
+	// HivelocityClusterReadyCondition reports on whether the Hivelocity cluster is in ready state.
+	HivelocityClusterReadyCondition clusterv1.ConditionType = "HivelocityClusterReady"
```

Comment:
do we still use this? The two reasons also sound related to CredentialsAvailable. Can you check?

Reply:
https://github.com/hivelocity/cluster-api-provider-hivelocity/blob/1b3e55ef94fdf8c74ca77f9ccc3e00bd9bd31e3a/controllers/hivelocitycluster_controller.go#L210 it is only used here, as a `MarkTrue()`, also the reasons associated with it are used by a `CredentialsAvailableCondition` MarkFalse(), what do you suggest?

Reply:
remove all of it